### DISCUSSION TITLE
faq/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#123] [Announcements]
# faq/

https://csmsapp.org/faq/

**Comment by wshanmu** [comment]


**Reply by cindyuc** [reply]
非常有必要。值得一试的。

**Reply by xichenpan** [reply]
可以参考author guideline，大部分ai conference都只要求不主动宣传即可，作为申请材料是完全OK的。其他领域会严格一些。如果导师不同意提及，可以使用伪标题，写明under review，author写成1st author publication这种即可。
关于补充材料，PhD随时可以和POI update，对于ms，大部分学校是不支持update的，有一些学校可以通过portal update，比如yale。此外还可以和小秘发邮件，问一下是否愿意帮自己update cv即可

**Comment by TadejPogacar1** [comment]
提问：以申请北美phd或master为目标，分别是否推荐考雅思，替代托福进行提交？

**Reply by xichenpan** [reply]
@TadejPogacar1 感觉没有必要，基本所有学校都接受TOEFL成绩，而且TOEFL要求一般较低。有些项目TOEFL 100对应IELTS 7.5，考IELTS略亏

**Reply by TadejPogacar1** [reply]
> @TadejPogacar1 感觉没有必要，基本所有学校都接受TOEFL成绩，而且TOEFL要求一般较低。有些项目TOEFL 100对应IELTS 7.5，考IELTS略亏
只是亏一点点分而已吗？是否有较多（北美master/phd）项目不接受IELTS，或者设置极高的要求（如口语7.5），即事实上不接受？

**Comment by xichenpan** [comment]
https://opencs.app/program_info/

TadejPogacar1 ***@***.***> 于 2023年4月10日周一 11:20写道：

**Reply by TadejPogacar1** [reply]
美国有门槛不那么高（UIUC MSCS, CMU MSR等）的研究型硕士（对标A+/A/A-/B+)吗？

**Reply by xichenpan** [reply]
没有，美国研究型硕士本来就很少，有一些可以做研究的（但实际上还是self-funed，同学也基本都去找工了）比如TAMU MSCS UCI MSCS

**Comment by gzy1223** [comment]
Caltech 的MS EE算什么级别呢

**Reply by wshanmu** [reply]
感觉得算SS，每年陆本严格小于5个人，一般都是清北本科，加上Caltech本身偏理论，对研究经历应该比较看重

**Comment by TadejPogacar1** [comment]
极高的GRE（如335）和borderline附近的GRE（320-325）（意味着不一定会交）能对master申请产生多大的difference？前者带来的优势能否相当于百分制均分提高1-2分？

**Reply by xichenpan** [reply]
我的看法是在绝大多数项目不会

**Comment by SergioPerez-png** [comment]
请问如果参加过海外学期交换并能转学分回原学校，那么做WES认证时海外高校的成绩是否会被考虑进去？如果会，是不是意味着gpa普遍会高一些？

**Comment by TadejPogacar1** [comment]
ccf计算机期刊与同等级会议，哪个对申请的加成比较大？

**Comment by xichenpan** [comment]
我觉得是会议，不过当然不算pami这种journal

TadejPogacar1 ***@***.***> 于 2023年11月17日周五 09:31写道：

**Comment by GuoZheXinDeGuang** [comment]
想问问美国本科 有research经历 产出只有一篇三四作者（有可能arxiv不顶会）可以试试看phd申请吗
